presto-main/src/main/resources/webapp/query.html

```diff
@@ -174,6 +177,10 @@
     d3.select('#sessionCatalog').text(query.session.catalog);
     d3.select('#sessionSchema').text(query.session.schema);
     d3.select('#elapsedTime').text(query.queryStats.elapsedTime);
+    if (query.state == 'FINISHED' || query.state == 'FAILED') {
```

Comment:
This misses other states like ABORTED. Shouldnt we always render the end time if it exists?